src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -551,15 +558,18 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id, HR
     if (Logger::IsDebugEnabled())
     {
         Logger::Debug("ModuleLoadFinished: ", module_id, " ", module_info.assembly.name, " AppDomain ",
-                      module_info.assembly.app_domain_id, " ", module_info.assembly.app_domain_name, " | IsNGEN = ", (module_info.IsNGEN() ? "true" : "false"));
+                      module_info.assembly.app_domain_id, " ", module_info.assembly.app_domain_name,
+                      " | IsNGEN = ", (module_info.IsNGEN() ? "true" : "false"),
+                      " | IsDynamic = ", (module_info.IsDynamic() ? "true" : "false"),
+                      " | IsResource = ", (module_info.IsResource() ? "true" : "false"));
     }
```

Comment:
FYI, if `IsNGEN` and others are `bool` ,you can use `std::boolalpha` to express that the variable are boolean and the ouput should output true or false and `std::noboolalpha` to express that the next one are not to be considered as `bool`
https://en.cppreference.com/w/cpp/io/manip/boolalpha

Reply:
TIL, done in f18e9da